app/src/main/AndroidManifest.xml

```diff
@@ -102,7 +102,8 @@
       android:theme="@style/OppiaThemeWithoutActionBar" />
     <activity
       android:name=".app.settings.profile.ProfileResetPinActivity"
-      android:theme="@style/OppiaThemeWithoutActionBar" />
+      android:theme="@style/OppiaThemeWithoutActionBar"
+      android:label="@string/profile_reset_pin_activity_label"/>
```

Comment:
Instead of `label` use suffix `title`

Also, add space before `/>`

Reply:
Okay,
Can you please explain the difference between the usage of label and title?
Also, there are some activities which are using `label` instead of `title` in a similar manner (the reference PR itself has `android:label` ), should we change that as well introducing an issue?
Why are we referencing it as a label in the issue title itself, when we are supposed to introduce a title?
Thank you for your help.